pkg/config/analysis/msg/messages.yaml

```diff
@@ -494,6 +494,17 @@ messages:
     - name: minimumVersion
       type: string
 
+  - name: "FutureUnsupportedGatewayAPIVersion"
+    code: IST0172
```

Comment:
nit: should we keep the original increasing order, or add a comment at the bottom to indicate that the next available is IST0173?